ui/app/(ai)/analyst/(tools)/compliances.ts

```diff
@@ -0,0 +1,55 @@
+import { tool } from "@langchain/core/tools";
+
+import {
+  aiGetComplianceOverview,
+  aiGetCompliancesOverview,
+} from "@/actions/lighthouse/compliances";
+import { aiGetComplianceFrameworks } from "@/lib/lighthouse/helperComplianceFrameworks";
+import {
+  getComplianceFrameworksSchema,
+  getComplianceOverviewSchema,
+  getCompliancesOverviewSchema,
+} from "@/types/lighthouse";
+
+export const getCompliancesOverviewTool = tool(
```

Comment:
Following up on this comment: https://github.com/prowler-cloud/prowler/pull/7508#discussion_r2052086824 

Could we please use the `/app` folder only for routing purposes?

Thank you!

Reply:
Moved all tools code to `/lib/lighthouse/tools/` directory